### ISSUE TITLE
Some Cypress integration tests failing during CI

### ISSUE THREAD

**Issue by ntijoh-daniel-berg** [itggot-TE4/Yabs#714]
### Description

Some Cypress integration tests failing during CI
- createTitleAndBooks.js          
- FirstPageTests.spec.js 
- updateTitle.js    

See https://github.com/itggot-TE4/Yabs/runs/4477863473?check_suite_focus=true 
and artifacts on https://github.com/itggot-TE4/Yabs/actions/runs/1561271949

### Expected behavior
The tests do not fail

## 🔬 To Reproduce
1. Push to github
2. Wait for integration tests to run
3. Se errors under actions -> integration -> select commit

**Comment by Linkaan** [comment]
`cy.get` is used with a too generic selector which returns an ambiguous result. This results in the wrong element being selected in the e2e tests. This could be solved by using a more specific selector:
https://github.com/itggot-TE4/Yabs/blob/3d45ad94f1bd94b73bae051c8b762826d1778692/frontend/cypress/integration/createTitleAndBooks.js#L20
Looking at the VMenu component this would seem to be a better selector:
```diff
- cy.get('div').contains('Kurslitteratur').click();
+ cy.get('.v-menu__content').contains('Kurslitteratur').click();
```

**Comment by ntijoh-daniel-berg** [comment]
@Linkaan 🤗 👍

**Comment by olantig** [comment]
Started working on this today but PR 761 didn't solve anything.

**Comment by Linkaan** [comment]
Couldn't help myself so I provided a fix for the issue here #764

Basically the definition of done is not strictly followed which involves making changes to tests if necessary ;)

**Comment by Linkaan** [comment]
One last test failing. An issue was created for it here #766
